paddle/pybind/CMakeLists.txt

```diff
@@ -5,4 +5,6 @@ if(WITH_PYTHON)
     ${GLOB_OP_LIB})
 endif(WITH_PYTHON)
 
-cc_binary(print_operators_doc SRCS print_operators_doc.cc DEPS ${GLOB_OP_LIB})
+if(WITH_DOC)
```

Comment:
Should also disable https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/scripts/docker/build.sh#L186 when WITH_DOC not enabled.

Reply:
Done in #6184